### PR TITLE
Update evaluate module (no ic)

### PR DESCRIPTION
**Pull request by roshankern**
This PR is ready for review!

In this PR, we complete evaluations for the no ic models/data.

**Note:** None of the LOIO notebooks have been completed in this PR. [get_LOIO_probabilities.py](https://github.com/WayScience/phenotypic_profiling_model/pull/50/files/55562b95f27c8d8b88035b3c72fb573c0837032c#diff-bb2a315ba6e162f9a46d29d2917dc577ab7734336829482f51c562b0df444f2b) has been setup and tested on only one model

### CODE REVIEW

File: 3.evaluate_model/README.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 In this module, we evaluate the final and shuffled baseline ML models.
 
 After training the models in [2.train_model](../2.train_model/), we use these models to predict the labels of the training and testing datasets and evaluate their predictive performance.
-Evaluations are done on each model for each combination of model type (final, shuffled baseline), feature type (CP, DP, CP_and_DP), and dataset (train, test).
+Evaluations are done on each model for each combination of model type (final, shuffled baseline), feature type (CP, DP, CP_and_DP), balance type (balanced, unbalanced), dataset type (ic, no_ic) and dataset (train, test).
```
Comment (gwaybio) [inline 3.evaluate_model/README.md]:
```suggestion
We evaluate each model for each combination of model type (final, shuffled baseline), feature type (CP, DP, CP_and_DP), balance type (balanced, unbalanced), dataset type (ic, no_ic) and dataset (train, test).
```

Active voice :)

File: 3.evaluate_model/scripts/nbconverted/F1_scores.py [hunk 2]
```diff
@@ -51,20 +49,28 @@
...
+    # determine model/feature type/balance/dataset type from model file name
     model_components = model_path.name.split("__")
-    # Older models only have 2 components, skip these
-    if len(model_components) == 2:
-        continue
     model_type = model_components[0]
     feature_type = model_components[1]
-    balance_type = model_components[2].replace(".joblib", "")
+    balance_type = model_components[2]
+    dataset_type = model_components[3].replace(".joblib", "")
```
Comment (gwaybio) [inline 3.evaluate_model/scripts/nbconverted/F1_scores.py:58]: can you add comments here describing what "dataset_type" means

File: 3.evaluate_model/scripts/nbconverted/get_LOIO_probabilities.py [hunk 3]
```diff
@@ -175,6 +165,10 @@
 
         # add tidy long data to compiled data
         compiled_LOIO_wide_data.append(test_cells_wide_data)
+        
+        break
```
Comment (gwaybio) [inline 3.evaluate_model/scripts/nbconverted/get_LOIO_probabilities.py:170]: not sure why these are added?

Comment (gwaybio) [reply 3.evaluate_model/scripts/nbconverted/get_LOIO_probabilities.py:170]: ooh, maybe you say in the PR comments that you haven't run this yet?

**Review by gwaybio — APPROVED** [review]
LGTM - only minor comments. I think the biggest one will be to add details on what you mean by "dataset_type"

File: 3.evaluate_model/scripts/nbconverted/F1_scores.py (hunk 2 above)
Comment (roshankern) [reply 3.evaluate_model/scripts/nbconverted/F1_scores.py:58]: Done!

**Pull request merged**